main/src/com/google/refine/RefineServlet.java

```diff
@@ -71,9 +71,6 @@
     static public String FULL_VERSION = "";
     static public String FULLNAME = "OpenRefine ";
 
-
-    static public final String AGENT_ID = "/en/google_refine"; // TODO: Unused?  Freebase ID
-    
     static final long serialVersionUID = 2386057901503517403L;
```

Comment:
I doubt this class ever gets serialized, but, technically, the `AGENT_ID` constant is part of the public API and removing it should probably change the `serialVersionUID`.

Reply:
@tfmorris Can you generate the new UID and submit a PR?